### ISSUE TITLE
Kivy Error during the lunch on Mac M1

### ISSUE THREAD

**Issue by Sowaky24** [kivy/kivy#7559]
**Software Versions**
* Python:3.9.5
* OS:Mac M1 Big Sur
* Kivy:2.0
* Kivy installation method:pip install

**Describe the bug**

Hello,

When I run the kivy demo, it gives me a kivy.clock error (see the screen).
pip list : 
Package     Version
----------- ---------
certifi     2021.5.30
chardet     4.0.0
docutils    0.17.1
idna        2.10
Kivy        2.0.0
Kivy-Garden 0.1.4
pip         21.1.2
Pygments    2.9.0
requests    2.25.1
setuptools  56.0.0
urllib3     1.26.5
wheel       0.36.2

thanks for your help.
Regards,

**Code and Logs and screenshots**

[https://gyazo.com/d224cc2a84774b3bf74ad223f581294e](url)

PycharmProjects/test3/main.py
[INFO   ] [Logger      ] Record log in /Users/williamgonzalez/.kivy/logs/kivy_21-06-22_10.txt
[INFO   ] [Kivy        ] v2.0.0
[INFO   ] [Kivy        ] Installed at "/Users/williamgonzalez/PycharmProjects/test3/venv/lib/python3.9/site-packages/kivy/__init__.py"
[INFO   ] [Python      ] v3.9.5 (v3.9.5:0a7dcbdb13, May  3 2021, 13:05:53) 
[Clang 12.0.5 (clang-1205.0.22.9)]
[INFO   ] [Python      ] Interpreter at "/Users/williamgonzalez/PycharmProjects/test3/venv/bin/python"
[ERROR  ] [Clock       ] Unable to import kivy._clock. Have you perhaps forgotten to compile kivy? Kivy contains Cython code which needs to be compiled. A missing kivy._clock often indicates the Cython code has not been compiled. Please follow the installation instructions and make sure to compile Kivy

**Comment by pctj101** [comment]
since it says "A missing kivy._clock often indicates the Cython code has not been compiled. "

maybe.... install cython?

something like... (not that I know the exact syntax, but you can search around)

brew install cython
pip3 install Cython

**Comment by paisleypark3121** [comment]
Salme issue with M1 McBook Air. I tried with explicit brew install cython but the issue persists:

ERROR  ] [Clock       ] Unable to import kivy._clock. Have you perhaps forgotten to compile kivy? Kivy contains Cython code which needs to be compiled. A missing kivy._clock often indicates the Cython code has not been compiled. Please follow the installation instructions and make sure to compile Kivy

any solution for this issue?

**Comment by TechVader** [comment]
Ran into this same issue. Finally decided to build from source when I found an answer that fixed this issue.

I originally installed it via the following line:
`python -m pip install 'kivy[full]' kivy_examples`

And that's when I got the error when trying to run a simple Kivy app.

Then I found the following line when I gave up hope:
`python -m pip install "kivy[full] @ https://github.com/kivy/kivy/archive/master.zip"`

After that, IT WORKED!

Then I uninstalled everything and tried it again with the following line:
`python -m pip install "kivy[full] @ https://github.com/kivy/kivy/archive/stable.zip"`

And yet again, it DID NOT work.

So again I ran:
`python -m pip install "kivy[full] @ https://github.com/kivy/kivy/archive/master.zip"`

and IT WORKED!
